test-e2e/src/bin/web.rs

```diff
@@ -1,4 +1,64 @@
-// TODO: copying pkg that includes wasm files from mopro-wasm
+use std::{path::Path, process::Command};
+
 fn main() {
-    println!("Perform wasm files from mopro-wasm");
```

Comment:
I am thinking about this
can it be like binaries in iOS and android
e.g.
```rs
mopro_ffi::app_config::ios::build();
```

and we can maintain a crate called `mopro_wasm`
and it just call the 
```rs
mopro_wasm::app_config::web::build();
```

so the e2e apps doesn't have to maintain the logic of `mopro_wasm` build (when we upgrade the script)
so we need a file in `mopro-wasm/src/app_config` like in `mopro-ffi/src/app_config`
to maintain the logic